src/middleware/validation.js

```diff
@@ -80,4 +76,23 @@ export default {
     }
     next();
   },
+  signInValidation(req, res, next) {
+    const loginSchema = Joi.object().keys({
+      email: validationRules.email,
+      password: validationRules.loginPassword
+    });
+    const options = {
+      language: {
+        key: '{{key}} '
+      }
+    };
```

Comment:
I see that the options variable has been repeated several times throughout this file. It would be better to create it once somewhere and then reuse it where ever you please. that will definately avoid the duplication that's happening here.

Reply:
Done